src/pages/loading-waiting-states/progress-bar/index.tsx

```diff
@@ -255,44 +252,38 @@ export const ProgressBar = (): JSX.Element => {
                 <List data-cy={'list-content'} disablePadding component="nav" className={classes.placementOfList}>
                     {fileUploadList.map(
                         (item, i): JSX.Element => (
-                            <div key={`itemKey${item.id}`} className={classes.infoList}>
+                            <div key={`itemKey${item.id}`} className={classes.fileUploadItem}>
                                 <Snackbar
-                                    classes={{ root: classes.snackbar }}
-                                    open={true}
+                                    classes={{
+                                        root: classes.snackbarRoot,
+                                        anchorOriginBottomCenter: classes.bottomCenter,
```

Comment:
instead of overriding the center-bottom styles...can we achieve what we want by setting the anchor point for the snackbar? https://material-ui.com/components/snackbars/#positioned-snackbars